src/write_mongodb.c

```diff
@@ -162,7 +171,7 @@ static int wm_initialize(wm_node_t *node) /* {{{ */
     uri_length = strlen(format_string) + strlen(node->user) +
                  strlen(node->passwd) + strlen(node->host) + 5 +
                  strlen(node->db) + 1;
-    if ((uri = malloc(sizeof(char) * uri_length)) == NULL) {
+    if ((uri = calloc(uri_length, sizeof(char))) == NULL) {
```

Comment:
`calloc(1, uri_length)` reads as "allocate 1 element of size uri_length for the url", whereas `calloc(uri_length, 1)` reads as "allocate uri_length elements of size 1". It's hard to explain the difference, and in practice it doesn't matter, but we use the first form throughout the codebase.